### ISSUE TITLE
[BUG] ReferenceError in profile release

### ISSUE THREAD

**Issue by xvw** [ocsigen/js_of_ocaml#1520]
**Describe the bug**
Still working on my personal site (whose repo is here, specifically on the branch that's causing problems) : https://github.com/xvw/capsule/tree/move-to-nightmare

I'm using a version pinned to master and the project works without the release profile, but as soon as I use the `release` profile, I get this error:

```
02:25:19.907 Layout was forced before the page was fully loaded. If stylesheets are not yet loaded this may cause a flash of unstyled content. node.js:401
02:25:20.022 mounting Connect in #dapp-transfer capsule_client.bc.js:9844:261
02:25:20.025 ReferenceError: can't access lexical declaration 'b' before initialization
    e http://localhost:8888/js/capsule_client.bc.js:411
    d http://localhost:8888/js/capsule_client.bc.js:2638
    au http://localhost:8888/js/capsule_client.bc.js:2646
    au http://localhost:8888/js/capsule_client.bc.js:2638
    loadOrCreateBeaconSecret http://localhost:8888/js/capsule_client.bc.js:2690
    j http://localhost:8888/js/capsule_client.bc.js:2661
    promise callback*f http://localhost:8888/js/capsule_client.bc.js:2667
    b9 http://localhost:8888/js/capsule_client.bc.js:2667
    b9 http://localhost:8888/js/capsule_client.bc.js:2660
    loadOrCreateBeaconSecret http://localhost:8888/js/capsule_client.bc.js:2685
    initSDK http://localhost:8888/js/capsule_client.bc.js:2682
    b9 http://localhost:8888/js/capsule_client.bc.js:2667
    b9 http://localhost:8888/js/capsule_client.bc.js:2660
    initSDK http://localhost:8888/js/capsule_client.bc.js:2680
    ae http://localhost:8888/js/capsule_client.bc.js:2676
    Se http://localhost:8888/js/capsule_client.bc.js:2807
    <anonymous> http://localhost:8888/js/capsule_client.bc.js:6060
    mountTransfer http://localhost:8888/js/capsule_client.bc.js:12442
    dn http://localhost:8888/js/capsule_client.bc.js:6648
    m2 http://localhost:8888/js/capsule_client.bc.js:7170
    <anonymous> http://localhost:8888/dapps/transfer/:55
    e http://localhost:8888/js/capsule_client.bc.js:9830
    h http://localhost:8888/js/capsule_client.bc.js:7715
    pn http://localhost:8888/js/capsule_client.bc.js:9719
    h http://localhost:8888/js/capsule_client.bc.js:7715
    hH http://localhost:8888/js/capsule_client.bc.js:9673
    pn http://localhost:8888/js/capsule_client.bc.js:9718
    c9 http://localhost:8888/js/capsule_client.bc.js:9749
    A http://localhost:8888/js/capsule_client.bc.js:9830
    a http://localhost:8888/js/capsule_client.bc.js:6655
    h http://localhost:8888/js/capsule_client.bc.js:7715
    x http://localhost:8888/js/capsule_client.bc.js:9832
    h http://localhost:8888/js/capsule_client.bc.js:7715
    f http://localhost:8888/js/capsule_client.bc.js:9697
    h http://localhost:8888/js/capsule_client.bc.js:7715
    d http://localhost:8888/js/capsule_client.bc.js:9665
    pd http://localhost:8888/js/capsule_client.bc.js:9668
    pd http://localhost:8888/js/capsule_client.bc.js:9668
    dz http://localhost:8888/js/capsule_client.bc.js:9672
    h http://localhost:8888/js/capsule_client.bc.js:7715
    pf http://localhost:8888/js/capsule_client.bc.js:9669
    dz http://localhost:8888/js/capsule_client.bc.js:9672
    pg http://localhost:8888/js/capsule_client.bc.js:9677
    n http://localhost:8888/js/capsule_client.bc.js:9836
    d http://localhost:8888/js/capsule_client.bc.js:9838
    dn http://localhost:8888/js/capsule_client.bc.js:6648
    aEI http://localhost:8888/js/capsule_client.bc.js:7165
    aiH http://localhost:8888/js/capsule_client.bc.js:9840
    dn http://localhost:8888/js/capsule_client.bc.js:6648
    m2 http://localhost:8888/js/capsule_client.bc.js:7170
    <anonymous> http://localhost:8888/dapps/transfer/:101
```

And here you can find the generated bundle : https://gist.github.com/xvw/128e5c5263502a0205cd8290cc019712

**Expected behavior**
The behaviour of released bundle should be the same.

**Versions**
Everything is described here https://github.com/xvw/capsule/blob/move-to-nightmare/capsule.opam

**Comment by xvw** [comment]
The online version (compiled using `--profile=release`) seems working so I close ATM.
Thanks and sorry for the noise.

**Comment by hhugo** [comment]
How am I suppose to trigger the issue with https://xvw.lol/dapps/transfer/ ?

**Comment by hhugo** [comment]
Should be fixed by #1521
